docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -1213,6 +1217,7 @@ Implements websocket protocol handshake according to [RFC 6455](http://tools.iet
 
 Query Parameters:
 
+-   **detacheys** – Specify the escape key sequence used to detach a container.
```

Comment:
`Specify a custom key sequence for detaching a container. Format is ctrl-<value> where <value> is one of: a-z, @, ^, [, ``, or _`